packages/examples/iotcc_edge_system_stats_with_filter.py

```diff
@@ -63,11 +68,11 @@ def read_cpu_utilization(sample_duration_sec=1):
 
 
 def read_disk_usage_stats():
-    return disk_usage_filter.filter(round(disk_stat.disk_reads_writes('sda')[0], 2))
+    return disk_usage_filter.filter(round(disk_stat.disk_reads_writes(disk_name)[0], 2))
 
 
 def read_network_bytes_received():
-    return net_usage_filter.filter(round(net_stat.rx_tx_bytes('ens33')[0], 2))
+    return net_usage_filter.filter(round(net_stat.rx_tx_bytes(network_interface)[0], 2))
 
 
 class PackageClass(LiotaPackage):
```

Comment:
All the above files including this shouldn't be part of this PR.  Please delete these or raise a separate PR